runbot_gitlab_minimal/models/runbot_repo.py

```diff
@@ -113,6 +114,13 @@ def github(self, cr, uid, ids, url, payload=None, ignore_errors=False,
                         json = json[0]
                         json['head'] = {'ref': json['target_branch']}
                         json['base'] = {'ref': json['source_branch']}
+                    if '/commits/' in url:
+                        url = _get_url('/users?search=%s' %
+                                       json['committer_email'], repo.base)
+                        response = session.get(url)
+                        response.raise_for_status()
+                        json['author'] = response.json()[0]['username']
```

Comment:
There are cases where the committer is not a user of gitlab
A example is the following commit: https://github.com/Vauxoo/odoo-peru/commit/ae11e24d2d678b327f64637172a88b92eec7526d where the committer `Arturo Flores` don't have github account and we had a TypeError fixed from https://github.com/Vauxoo/runbot-addons/pull/95

Could you review if we will have a special exception for gitlab?